src/Bicep.Core.UnitTests/Assertions/JTokenAssertionsExtensions.cs

```diff
@@ -13,7 +13,7 @@ namespace Bicep.Core.UnitTests.Assertions
 {
     public static class JTokenAssertionsExtensions
     {
-        public static JTokenAssertions Should(this JToken instance)
+        public static JTokenAssertions Should(this JToken? instance)
```

Comment:
>JToken [](start = 51, length = 6)

I'm guessing this led to usages of `!` to suppress the nullness error all over the place. Worth going back to clean up?